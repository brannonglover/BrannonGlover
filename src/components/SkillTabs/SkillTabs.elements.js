import styled from 'styled-components/macro';

export const SkillTabsWrapper = styled.div`
  display: flex;
  flex-wrap: wrap;
`;

export const SkillTab = styled.div`
  padding: .5em 1em;
  margin: .2em .2em;
  background-color: #6d6b78;
  border-radius: 5px;
  color: #fff;
`;