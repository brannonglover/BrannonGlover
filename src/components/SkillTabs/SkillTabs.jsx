import React from 'react';
import { SkillTabsWrapper, SkillTab } from './SkillTabs.elements';
import data from '../../assets/info.json';

export default function SkillTabs() {
  return (
    <SkillTabsWrapper>
      {data.technical_skills.map(skill => (
        <SkillTab key={skill}>{skill}</SkillTab>
      ))}
    </SkillTabsWrapper>
  )
}