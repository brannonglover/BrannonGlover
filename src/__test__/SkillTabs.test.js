import React from 'react';
import { render } from '@testing-library/react';
import SkillTabs from '../components/SkillTabs/SkillTabs';

test('Render SkillTabs component', () => {
  const { asFragment } = render(<SkillTabs />);

  expect(asFragment()).toMatchSnapshot();
});