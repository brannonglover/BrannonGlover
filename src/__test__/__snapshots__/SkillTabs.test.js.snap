// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render SkillTabs component 1`] = `
<DocumentFragment>
  <div
    class="SkillTabselements__SkillTabsWrapper-mdholk-0 hxGMPF"
  >
    <div
      class="SkillTabselements__SkillTab-mdholk-1 bjlGyP"
    >
      React
    </div>
    <div
      class="SkillTabselements__SkillTab-mdholk-1 bjlGyP"
    >
      React Hooks
    </div>
    <div
      class="SkillTabselements__SkillTab-mdholk-1 bjlGyP"
    >
      Context
    </div>
    <div
      class="SkillTabselements__SkillTab-mdholk-1 bjlGyP"
    >
      Redux
    </div>
    <div
      class="SkillTabselements__SkillTab-mdholk-1 bjlGyP"
    >
      TypeScript
    </div>
    <div
      class="SkillTabselements__SkillTab-mdholk-1 bjlGyP"
    >
      JavaScript
    </div>
    <div
      class="SkillTabselements__SkillTab-mdholk-1 bjlGyP"
    >
      REST
    </div>
    <div
      class="SkillTabselements__SkillTab-mdholk-1 bjlGyP"
    >
      CSS
    </div>
    <div
      class="SkillTabselements__SkillTab-mdholk-1 bjlGyP"
    >
      SASS
    </div>
    <div
      class="SkillTabselements__SkillTab-mdholk-1 bjlGyP"
    >
      JSON
    </div>
    <div
      class="SkillTabselements__SkillTab-mdholk-1 bjlGyP"
    >
      styled-components
    </div>
    <div
      class="SkillTabselements__SkillTab-mdholk-1 bjlGyP"
    >
      HTML
    </div>
  </div>
</DocumentFragment>
`;
